README.md

```diff
@@ -10,12 +10,51 @@ Send a HTML or Mustache template and recieve a PDF stream as the response.
 
 ## Install and start
 
-### Node App
+### Docker container
+
+Navigate to quay.io/ukhomeofficedigital/html-pdf-converter to find latest tagged version this will be the image that docker will pull.
```

Comment:
Docker will pull any image tag that you specify, so it might be better to say **to find the latest tagged version. Docker will pull whichever version you specify. Use the latest tag unless you have a reason to pull an older image** (or something like that)

Reply:
> Docker will pull any image tag that you specify, so it might be better to say **to find the latest tagged version. Docker will pull whichever version you specify. Use the latest tag unless you have a reason to pull an older image** (or something like that)

The latest tag for Rhodine and I didn't work and I got this error: 

> $ docker pull quay.io/ukhomeofficedigital/html-pdf-converter  [16:20:54]
> Using default tag: latest
> Error response from daemon: manifest for quay.io/ukhomeofficedigital/html-pdf-converter:latest not found: manifest unknown: manifest unknown

It seems like the latest image for this microservice has a version tag,  maybe that's why latest doesn't work?

Reply:
sorry that's confusing - I mean 'Use the tag for the latest version present' rather than looking for the ':latest'.